src/cclib/io/moldenwriter.py

```diff
@@ -10,19 +10,28 @@
 import os.path
 
 from . import filewriter
+from .filewriter import MissingAttributeError
 from cclib.parser import utils
 
 
 class MOLDEN(filewriter.Writer):
     """A writer for MOLDEN files."""
 
+    def _check_required_attributes(self, required):
```

Comment:
I moved this function to the base class. For now, I am avoiding passing the list of required attributes to base class though.